ISHLogDNAService/ISHLogDNAService.m

```diff
@@ -94,7 +97,10 @@ + (instancetype)sharedInstance {
 
     dispatch_once(&pred, ^{
         sharedInstance = [[self alloc] init];
+        
+#if TARGET_OS_IOS || TARGET_OS_TV
         sharedInstance.enabled = [[ASIdentifierManager sharedManager] isAdvertisingTrackingEnabled];
```

Comment:
I think defaulting to `YES` will mean nobody will care and just send it. Defaulting to `NO` will ensure people will at least think about it.